.gitignore

```diff
@@ -1,3 +1,5 @@
 *~
 playbooks/group_vars/all
 playbooks/db-change-record-set.json
+playbooks/roles/creategroups/files/s3-policy.json
+playbooks/s3-policy.json
```

Comment:
Why are those 2 files in .gitignore? I would guess one of them could be but both?